data-otservbr-global/lib/core/storages.lua

```diff
@@ -1573,6 +1573,8 @@ Storage = {
 			WarriorSkeleton = 52146,
 			DragonCounter = 52147,
 		},
+		QuestLine = 52148,
```

Comment:
if these storages are from the same quest, I think it should be better to be grouped as array storage. example:
TheLostBrother = {
   QuestLine = 52148,
   TheLostBrother = 52149
}

Reply:
Actually, the Questline is not from the quest "The Lost Brother" but from the quest "Adventurer's Guild".

"The Lost Brother" is a subquest of the Adventurer's Guild. I used this "Questline" as a way to open the Quest Adventurer's Guild, so that these subquests can be displayed. (following the pattern of other quests that are similar and have the same behavior)